aospy/calc.py

```diff
@@ -470,7 +468,7 @@ def _compute_full_ts(self, data, monthly_mean=False, zonal_asym=False):
                                                  self.end_date)
             )
             if self.dtype_out_vert == 'vert_av':
-                full_ts *= (grav.value / self._to_desired_dates(self._ps_data))
+                full_ts *= (GRAV_EARTH.data / self._to_desired_dates(self._ps_data))
```

Comment:
Here and elsewhere the constants are used, we should not need to use the `data` attribute.  In other words I think the following should be OK:
```python
full_ts *= (GRAV_EARTH / self._to_desired_dates(self._ps_data))
```